entr.1

```diff
@@ -83,72 +82,80 @@ A process group is created to prevent shell scripts from masking signals.
 waits for the
 .Ar utility
 to exit to ensure that resources such as sockets have been closed.
-Control of the TTY is not transferred the child process.
+Control of the TTY is not transferred to the child process.
 .It Fl s
-Evaluate the first argument using the interpreter specified by the
-.Ev SHELL
-environment variable.
-If stdout is a TTY, the name of the shell and exit code is printed after each
-invocation.
+Evaluate the first argument using the interpreter specified by
+.Ev SHELL .
+If standard output is a TTY, the name of the interpreter and exit code is
+printed after each invocation.
 .El
 .Pp
 The first argument named
-.Ar /_
+.Dq Ar /_
 is replaced with the absolute path of the first file to trigger an event.
 The first file under watch is used as the default.
 If the
 .Fl s
-option is used, the name of the first file to triggler an event can be read from
+option is used, the name of the first file to trigger an event can be read from
 .Va $0 .
-.Sh COMMANDS
+.Ss Commands
 .Nm
 listens for keyboard input and responds to the following commands:
-.Bl -tag -width Ic
-.It SPACE
-Execute the utility immediately.
+.Bl -tag -width <space>
+.It Aq Cm space
+Execute
+.Ar utility
+immediately.
 If the
 .Fl r
 option is set this will terminate and restart the child process as if a file
 change event had occurred.
-.It q
-Quit; equivalent pressing Ctrl\-C.
+.It Cm q
+Quit
+.Nm .
+Equivalent to pressing
+.Aq Cm control-c
+or sending
+.Dv SIGINT .
```

Comment:
True, but I think this is implied by Ctrl-C

Reply:
Better to be explicit in this case I think.